lib/mongoose_push/telemetry/handler.ex

```diff
@@ -0,0 +1,42 @@
+defmodule MongoosePush.Telemetry.Handler do
+  @moduledoc """
+  Module responsible for handling telemetry events, in future should be integrated with Prometheus
```

Comment:
Maybe let's not use `Prometheus` name explicitly, as we haven't officially decided that, and also, this doesn't actually matter.